zigbee2mqtt/DOCS.md

```diff
@@ -17,14 +37,15 @@ Enable `ingress` to have the frontend available in your UI: **Supervisor → Das
 
 # Configuration
 The configuration closely mirrors that of `zigbee2mqtt` itself, with a couple of key differences:
-1. Home Assistant requires add-on configuration in JSON format, rather than YAML. If you don't understand the difference, you can use a YAML-to-JSON converter.
+1. Home Assistant requires add-on configuration in JSON format, rather than YAML if you are editing this addon's configuration.yaml directly. If you are using the Home Assistant UI, then yaml is required. If you don't understand the difference, you can use a YAML-to-JSON converter.
 2. An additional top-level `data-path` option is required. Set this to the path where you would like the add-on to persist data. Defaults to `/share/zigbee2mqtt`. Note that both `config` and `share` directories are mapped into the container (read-write) and are available to you.
 3. If you are using groups or device-specific settings, you must use seperate files, and provide the paths to these files in their corresponding config options as described by the zigbee2mqtt docs. This is due to a limitation Home Assistant places on nested config levels.
 
 See the [zigbee2mqtt configuration docs](https://www.zigbee2mqtt.io/information/configuration.html) for a complete description of available options. If you're not sure if a new option is supported, check to see if it is included in this add-on's default configuration. If not, you can open an issue to add support for it.
 
 # Notes
 - Depending on your configuration, the MQTT server config may need to include the port, typically `1883` or `8883` for SSL communications. For example, `mqtt://core-mosquitto:1883` for Home Assistant's Mosquitto add-on.
+- A `user` and a `password` must be provided in the `mqtt` section of the settings. Failing to provide valid credentials or using anonymous authentication against the broker may lead to failure in configuring new devices.
```

Comment:
I think we don't need this section. (zigbee2mqtt/hassio-zigbee2mqtt)[https://github.com/zigbee2mqtt/hassio-zigbee2mqtt/blob/dev/common/rootfs/etc/cont-init.d/zigbee2mqtt.sh#L11] already support user & password & server discovery and it will get merged here at some point.